src/components/AddRepoForm/AddRepoForm.js

```diff
@@ -0,0 +1,48 @@
+import React, { useState } from 'react';
+import { useSelector, useDispatch } from 'react-redux'
+import { getRepository } from '../../actions'
+import { Form, InputGroup, FormControl, Button } from 'react-bootstrap';
+import './AddRepoForm.scss'
+
+const AddRepoForm = () => {
+  const [repoName, setRepoName] = useState('')
+  const error = useSelector(state => state.repositories.error)
+  const dispatch = useDispatch()
+
+  const handleSubmit = (e) => {
+    e.preventDefault()
+    dispatch(getRepository(repoName))
+    setRepoName('')
```

Comment:
I think you should clear input only after successful submit, if you get error you should add ability to make changes to already added text